textbook/12/1/uniform.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>Is there an outside reference (like maybe the stats book Dan wanted to refer students to for the probability lecture) that explains this a bit more? It's a bit out of context and left field. Like, how did one ascertain these mathematical definitions of the mean and variance?</p><p> </p><p>Is it assumed one would know this if they have already studied/familiar with the Law of Large Numbers? If so, then I guess that reference you mention would rectify the above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>